src/restconfig/src/main/java/org/geoserver/rest/catalog/StyleController.java

```diff
@@ -225,7 +225,7 @@ private UriComponents getUriComponents(String name, String workspace,
         return uriComponents;
     }
 
-    @GetMapping(path = {"/styles/{styleName}", "/workspaces/{workspaceName}/styles/{styleName}"})
+    @GetMapping(path = {"/styles/{styleName}", "/workspaces/{workspaceName}/styles/{styleName}"}, produces = {MediaType.ALL_VALUE})
```

Comment:
Nope, you did not. The test was not doing GET requests. I added some in this pull request.